src/resmom/prolog.c

```diff
@@ -378,7 +392,7 @@ int   pe_io_type;
 	sprintf(action_name, "pbs_pelog%d_%d", which, _getpid());
 	pelog_handle = CreateJobObject(NULL, action_name);
 
-	if (pelog_handle == INVALID_HANDLE_VALUE) {
+	if ((pelog_handle == INVALID_HANDLE_VALUE) || (pelog_handle == NULL)) {
```

Comment:
CreateJobObject() should either return NULL or INVALID_HANDLE_VALUE. what does the documentation say?

Reply:
> CreateJobObject() should either return NULL or INVALID_HANDLE_VALUE. what does the documentation say?

In the documentation, it is written that return is NULL only. But I kept INVALID_HANDLE_VALUE, because this check was there already and didn't want to change it. This check is there in other places of the code too.